README.md

```diff
@@ -148,7 +148,34 @@ Once you have created the parameter, you must remove the variable `runners_token
 
 Finally, the runner still supports the manual runner creation. No changes are required. Please keep in mind that this setup will be removed in future releases.
 
-### Auto Scaling Group Instance Termination
+### Auto Scaling Group
+
+#### Scheduled scaling
+
+When `enable_schedule=true`, the `schedule_config` variable can be used to scale the Auto Scaling group. 
+
+Scaling may be defined with one `scale_out` scheduled action and/or one `scale_in` scheduled action.
+
+For example:
+```hcl
+  module "runner" {
+    ...
+    enable_schedule = true
+    schedule_config = {
+      # Configure optional scale_out scheduled action
+      scale_out_recurrence = "0 8 * * 1-5"
+      scale_out_count      = 1 # Default for min_size, desired_capacity and max_size
+      # Override using: scale_out_min_size, scale_out_desired_capacity, scale_out_max_size
+
+      # Configure optional scale_in scheduled action
+      scale_in_recurrence  = "0 18 * * 1-5"
+      scale_in_count       = 0 # Default for min_size, desired_capacity and max_size
+      # Override using: scale_out_min_size, scale_out_desired_capacity, scale_out_max_size
```

Comment:
question: Expected a `scale_out` when leaving the office at 6pm and a `scale_in` when coming back at 8am. Have they been swapped by accident?

Reply:
Thanks for the feedback.

They look correct, same as before. 
`scale_out` at the start of the day (create more instances) and `scale_in` at end of the day (terminate).

It does get easily confused. I tried to move the code slightly to clarify / avoid confusion but the logic is identical otherwise.

Maybe clearer if I change from:
`# Configure optional scale_out scheduled action`
to
`# Configure optional scale_out scheduled action`
`# to create instances at the start of my work day`

and
`# Configure optional scale_in scheduled action`
`# to terminate instances at the end of my work day`

What do you think @kayman-mk